PowerEditor/src/MISC/md5/md5Dlgs.cpp

```diff
@@ -334,6 +341,27 @@ void HashFromTextDlg::setHashType(hashType hashType2set)
 	_ht = hashType2set;
 }
 
+LRESULT HashFromTextDlg::HashTextEditRunProc(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
+{
+	switch (message)
+	{
+		case WM_CHAR:
+		{
+			// Select All if Ctrl + A
+			if (wParam == 1)
+			{
+				::SendDlgItemMessage(_hSelf, IDC_HASH_TEXT_EDIT, EM_SETSEL, 0, -1);
+				return TRUE;
+			}
+			break;
+		}
+
+		default:
+			break;
+	}
+	return _HashTextEditProc(hwnd, message, wParam, lParam);
```

Comment:
Perhaps, `return ::CallWindowProc(_HashTextEditProc, hwnd, message, wParam, lParam);` ?

Reply:
Agree. It's more conventional.
Change it and test it. If it works (we never know), commit it and push it.

Reply:
It works. :)